ui/src/app/job-details/tasks/tasks.component.ts

```diff
@@ -63,6 +61,14 @@ export class TaskDetailsComponent implements OnInit, OnChanges {
   getResourceFileName(url: string): string {
     return ResourceUtils.getResourceFileName(url);
   }
+
+  getTimingUrl(serverUrl: string, jobId: string): string {
+    if (serverUrl) {
+      return serverUrl + jobId + '/timing';
+    } else {
+      return '/jobs/' + jobId;
```

Comment:
Ideally this else should return something like `#` or `javascript:void(0);`, but for the former one from Angular2 changed the behavior from "don nothing" to "App root", and for latter one Angular automatically redirects it to `unsafe:javascript:void(0);` for security issues. Other alternatives require more work on module's configs, so I make it just refresh the current page if the `serverUrl` is '' (mostly for dsub)